src/pages/minjeong/Login/loginMj.scss

```
@@ -0,0 +1,77 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: 'Noto Sans KR', sans-serif;
+  background-color: whitesmoke;
+}
+
+.login {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 700px;
+  height: 760px;
+  border: 1px solid lightgray;
+
+  .westagram {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

```suggestion
  display: flex;
  justify-content: center;
  align-items: center;
}

.login {
  display: flex;
  justify-content: center;
  align-items: center;
  

  .westagram {
    display: flex;
    justify-content: center;
    align-items: center;
```
display: flex;
justify-content: center;
align-items: center;
를
@mixin flexCenter {
  display: flex;
  justify-content: center;
  align-items: center;
}
mixin을 사용해도 좋을 것 같아요!